openthread/Kconfig

```
@@ -24,4 +26,79 @@ config OPENTHREAD_LIBRARY_1_1
 
 endchoice
 
+# OpenThread configuration features sets from Nordic Semiconductor
+
+choice
+	prompt "Nordic library features set"
+	default OPENTHREAD_NORDIC_LIBRARY_NONE
+	help
+	  This option selects the Thread stack features set used for building the precompiled libraries.
+
+config OPENTHREAD_NORDIC_LIBRARY_NONE
+	bool "None"
+	help
+	  Do not enable any specific set of OpenThread features.
```

Would this be more of `Custom` configuration ?
```suggestion
config OPENTHREAD_NORDIC_LIBRARY_CUSTOM
	bool "Custom OpenThread configuration"
	help
	  Custom configuration of OpenThread features.
```